fmriprep/cli/parser.py

```diff
@@ -586,7 +587,7 @@ def parse_args(args=None, namespace=None):
             config.nipype.plugin = _plugin
             config.nipype.plugin_args = plugin_settings.get("plugin_args", {})
             config.nipype.nprocs = config.nipype.plugin_args.get(
-                "nprocs", config.nipype.nprocs
+                "n_procs", config.nipype.nprocs
```

Comment:
What happens if `n_procs` is defined in the YAML file AND with `--nprocs`? The latter should take precedence (not saying that it worked already like this... put since you are at it, I guess we should get this right).

Reply:
I would say that CLI options should override the yaml.

Reply:
Yes, I meant that. Not sure I successfully said it.

Reply:
Oh you did. My brain is just failing.